cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -114,16 +121,26 @@ public static void beforeAll() {
 
         vertx = Vertx.vertx();
         sharedWorkerExecutor = vertx.createSharedWorkerExecutor("kubernetes-ops-pool");
+        
+        // Configure Cruise Control mock
+        serverPort = CruiseControl.REST_API_PORT;
+        tlsKeyFile = TestUtils.tempFile(".key");
+        tlsCrtFile = TestUtils.tempFile(".crt");
+
+        new MockCertManager().generateSelfSignedCert(tlsKeyFile, tlsCrtFile,
+            new Subject.Builder().withCommonName("Trusted Test CA").build(), 365);
```

Comment:
What is the performance impact on this? Working with real certs significantly slows down the tests, so you should preferebly not do it.

Reply:
This is executed one time per test class and it takes around 2 microseconds.